frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-exclusions-section-card/feature-flag-exclusions-table/feature-flag-exclusions-table.component.ts

```diff
@@ -13,7 +14,8 @@ import { TranslateModule } from '@ngx-translate/core';
   imports: [CommonDetailsParticipantListTableComponent, CommonModule, TranslateModule],
 })
 export class FeatureFlagExclusionsTableComponent {
-  @Input() dataSource$ = this.featureFlagService.selectFeatureFlagExclusions$;
+  listType$ = FEATURE_FLAG_PARTICIPANT_LIST_KEY.EXCLUDE;
```

Comment:
what is this line about?

Reply:
The `listType$` (either "include" or "exclude") is used to pass it to the child `common-details-participant-list-table` component so that the child knows what columns to display in the table.

Reply:
ooooh, it's the $ that is throwing me, that's the convention for observable $treams, can it just be `listType`? `listType` is maybe a little overloaded term though, it is also used to refer to the listType of Segment / Individual / Group.... list. Maybe `tableType`? or something

Reply:
Got it, I will make that change.